defaults/main.yml

```diff
@@ -211,3 +211,7 @@ prometheus_alert_rules:
     for: 10m
     labels:
       severity: warning
+
+prometheus_rules_files_extension: "rules"
+prometheus_rules_files:
```

Comment:
Such customizability will cause additional maintenance burden. It is much easier to list 4 extension types and include them all instead.